src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const element = document.body.children[0];
+const collectionOfNumbers = element.querySelectorAll('.population');
+const arrayOfNumbers = [...collectionOfNumbers].map(number => {
```

Comment:
you don't copy your array when you use `.map`
```suggestion
const arrayOfNumbers = collectionOfNumbers.map(number => {
```